ocw/tests/test_metrics.py

```diff
@@ -208,41 +229,6 @@ def test_function_run(self):
         expected_result.fill(-300)
         np.testing.assert_array_equal(self.mean_bias.run(self.target_dataset,self.reference_dataset), expected_result)
 
-    def test_function_run_abs(self):
-        '''Test mean bias function between reference dataset and target dataset with abs as True.'''
-        expected_result = np.zeros((5, 5), dtype=np.int)
-        expected_result.fill(300)
-        np.testing.assert_array_equal(self.mean_bias.run(self.reference_dataset, self.target_dataset, True), expected_result)
-
-
-class TestSpatialMeanOfTemporalMeanBias(unittest.TestCase):
```

Comment:
@MJJoyce I removed the 'absolute=True/False' key word option from metrics.TemporalMeanBias. So we do not need this test. Users can always use numpy.absolute function to properly handle missing values after calculating biases.